### ISSUE TITLE
Namespaces cannot be found while the program runs normally

### ISSUE THREAD

**Issue by TheOnlylight** [microsoft/qsharp-compiler#434]
I use the vscode on Win10, with .net 3.1.4
the VScode reported red on the namespaces fro Q#, however the program managed to build and run properly ,this is so werid
![image](https://user-images.githubusercontent.com/24605645/82305080-c637ca80-99ef-11ea-9edb-6179de5c6141.png)
![image](https://user-images.githubusercontent.com/24605645/82305088-c932bb00-99ef-11ea-8ef8-8ed5f3d8ea4b.png)

**Comment by valdaarhun** [comment]
Hi. I too am facing this issue. The program works even though I get errors stating that the namespaces could not be found.
I am using vs code with .net 3.1.300. The version of the windows OS that I am using is 6.1.7601.
![Console](https://user-images.githubusercontent.com/39989901/82423443-60a41680-9aa1-11ea-99a7-7adb88662a94.PNG)
![Screen](https://user-images.githubusercontent.com/39989901/82423460-6863bb00-9aa1-11ea-8ac8-570880ee0d5f.PNG)
The screenshot below shows the contents of the Q# Language Extension tab of the Output pane.
![Output](https://user-images.githubusercontent.com/39989901/82423481-70235f80-9aa1-11ea-98af-566569183fbe.PNG)

**Comment by bettinaheim** [comment]
Yes, the output clarifies what the issue is! It looks like the package that we are using for the design time build is older than the dotnet version you are using. We can update that with our next release. Thanks for pointing it out! 
@ricardo-espinoza It would be good if we could find a more permanent solution than having to explicitly reference and update the NuGet.Frameworks package.

**Comment by bettinaheim** [comment]
See also https://github.com/microsoft/MSBuildLocator/issues/86

**Comment by venkyms** [comment]
Same namespace issue occurs in mac version as well. Below are the details

```
Mac version : 10.15.5 
VS code version : 1.45.1
dotnet core version : 3.1.300
Q# extension version: 0.11.20042825
```

Error Message from the output window(Q# Language Extension)
```
[Error - 5:42:29 PM] MSBuild error in /usr/local/share/dotnet/sdk/3.1.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(59,5): The "ProcessFrameworkReferences" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'NuGet.Frameworks, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

File name: 'NuGet.Frameworks, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.ExecuteCore()
   at Microsoft.NET.Build.Tasks.TaskBase.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)
```

Is there any work arround for this issue, until this get patched ? Thanks.

**Comment by bettinaheim** [comment]
@venkyms I believe this issue should indeed be fixed with the June 4th release. 
Unfortunately it might temporarily reoccur until [this](https://github.com/microsoft/MSBuildLocator/issues/86) is fixed or we find an alternative. The workaround in the case of reoccurrence would be to not use the latest dotnet version until we've had a chance to update the package as part of our release.

**Comment by valdaarhun** [comment]
Hi. I would like to point out that after updating vscode the errors no longer appear and the program runs properly.

**Comment by abidart** [comment]
Hello everyone, I think this might be happening again. I am experiencing the same issue, on `MacOS Mojave 10.14.6` and `dotnet version 3.1.401`. 

Here is the log (with the paths to the project edited out)

```
[Info  - 8:46:39 PM] workspace folder: [edited]
[Error - 8:46:40 PM] MSBuild error in /usr/local/share/dotnet/sdk/3.1.401/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(59,5): The "ProcessFrameworkReferences" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'NuGet.Frameworks, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

File name: 'NuGet.Frameworks, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.ExecuteCore()
   at Microsoft.NET.Build.Tasks.TaskBase.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)


[Error - 8:46:40 PM] Failed to restore project [edited]
[Error - 8:46:40 PM] MSBuild error in /usr/local/share/dotnet/sdk/3.1.401/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(59,5): The "ProcessFrameworkReferences" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'NuGet.Frameworks, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

File name: 'NuGet.Frameworks, Version=5.7.0.7, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Microsoft.NET.Build.Tasks.ProcessFrameworkReferences.ExecuteCore()
   at Microsoft.NET.Build.Tasks.TaskBase.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)


[Error - 8:46:40 PM] Failed to resolve assembly references for project [edited]
Discovered Q# project [edited]
Loading project [edited]
Done loading project [edited]
```

**Comment by samarsha** [comment]
Hi @abidart, that looks like the same problem as #558. It should be fixed in the next release sometime this week. :) Hopefully, the fix is permanent this time and should continue to work with newer .NET Core versions.

**Comment by alfogo** [comment]
Hi, any update on this?. I'm on Quantum Dev Kit 0.14.2011.120240 with .net core sdk 3.1.405 and getting exactly same issue

**Comment by samarsha** [comment]
Hi @alfogo, this issue was fixed in August, and QDK 0.14.2011120240 should work with .NET Core 3.1.405 SDK. I'm not sure what's causing the problem you're seeing, unfortunately. Could you create a new GitHub issue for it?
